users/views.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import jwt
+import json
+
+from django.views  import View
+from django.http   import JsonResponse
+
+from users.models       import User, Bookmark
+from jobpostings.models import JobPosting
+from my_settings        import DATABASES, SECRET_KEY, ALGORITHM
+from utils              import authorization
+
+class KakaoLoginView(View):
+    def post(self, request):
+        access_token = request.headers["Authorization"]
+        headers      = ({"Authorization": f"Bearer {access_token}"})
+        url          = "https://kapi.kakao.com/v2/user/me"
+        response     = requests.get(url, headers=headers)
+        user         = response.json()
+
+        users, created = User.objects.get_or_create(
+                name          = user["kakao_account"]["profile"]["nickname"],
+                email         = user["kakao_account"]["email"],
+                profile_image = user["kakao_account"]["profile"]["profile_image_url"],
+                kakao_api_id  = user["id"]
+            )
```

Comment:
get_or_create을 사용할 때는 pk 같이 값이 변하지 않고, 데이터를 특정 지을 수 있는 것들로 가져와야합니다. 만약에 지금 넣은 속성 중 1개라도 다르면 완전히 다른 데이터로 취급합니다. 따라서 다음과 같이 작성해 주는 것이 좋습니다.
```suggestion
        users, is_created = User.objects.get_or_create(kakao_api_id  = user["id"])
        
        if is_created:          
            user.name          = user["kakao_account"]["profile"]["nickname"],
            user.email         = user["kakao_account"]["email"],
            user.profile_image = user["kakao_account"]["profile"]["profile_image_url"]
            user.save()
```

Reply:
수정했습니다.